internal/types/resource/resource.go

```diff
@@ -1,6 +1,8 @@
 package resource
 
-import "encoding/json"
+import (
+	"encoding/json"
+)
```

Comment:
this change could be dropped now that nothing is being added to this file

Reply:
Done